app/templates/components/event-table.hbs

```diff
@@ -0,0 +1,85 @@
+  <div class="row">
+    <div class="sixteen wide column">
+      <table class="ui tablet stackable celled table">
+        <thead>
+          <tr>
+            <th>Logo</th>
+            <th>Name</th>
+            <th>Date</th>
+            <th>Roles</th>
+            <th>Sessions</th>
+            <th>Speakers</th>
+            <th>Tickets</th>
+            <th>Public Url</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr class="centre aligned">
+            <td>
+              <div>
+                <img src="http://placehold.it/100x100" alt="Event logo">
+              </div>
+            </td>
+            <td>
+              <div>Sample Event</div>
+            </td>
+            <td>
+              <div>27-06-2017</div>
```

Comment:
mostly to ensure they were in separate lines so as to they don't clutter up the table.